website/versioned_docs/version-5.x/module-types.md

```diff
@@ -0,0 +1,51 @@
+---
+id: module-types
+title: single-spa microfrontend types
+sidebar_label: Microfrontend Types
+---
+
+# Concept: single-spa microfrontend types
+
+Single-spa has [different categories](./microfrontends-concept#types-of-microfrontends) of microfrontends. It is up to you where and how you use each of them. However, the single-spa core team has [recommendations](./recommended-setup/#applications-versus-parcels-versus-utility-modules).
```

Comment:
[The docusaurus docs](https://v2.docusaurus.io/docs/versioning/#linking-files) suggest putting in `.md` for relative links, but the extension is not here. Have you verified whether omitting the `.md` still works?

Reply:
Yeah it does work. I can add the `.md` if you think that will work better though.